### ISSUE TITLE
Release 0.1.1 contains wrong binary

### ISSUE THREAD

**Issue by SystemKeeper** [nextcloud/notify_push#17]
Downloaded via app store and tried to install with the setup wizard.

```
root@nextcloud:/var/www/nextcloud# apps/notify_push/bin/x86_64/notify_push  --version
notify_push 0.1.0
root@nextcloud:/var/www/nextcloud# su - www-data -s /bin/bash -c 'php7.3 /var/www/nextcloud/occ app:list' | grep push
  - notify_push: 0.1.1
```

Which fails this test:
https://github.com/nextcloud/notify_push/blob/2e6b23703f83a1abf24579d615747c0b0c0222b8/lib/SetupWizard.php#L80-L86

**Comment by grawlinson** [comment]
It'll probably be because the version in `Cargo.toml` still says `0.1.0`.

**Comment by SystemKeeper** [comment]
Makes sense. There is already some work for the self-test: https://github.com/nextcloud/notify_push/commit/272bf74b26d2a5c9cef17db7468686c87697d508

**Comment by icewind1991** [comment]
This has been fixed with 0.1.2
